openapi-integration-tests/src/test/resources/service/return/ballerina/response_has_inline_additional_properties.bal

```diff
@@ -21,17 +26,17 @@ public type StoreInventory04Response record {|
     User...;
 |};
 
-public type Inline_response_map200 record {
-    int id?;
+public type inline_response_200 record {|
+    string name?;
     int age?;
-};
+    int...;
+|};
 
 public type StoreInventory05Response record {|
     User...;
 |};
 
-public type Inline_response_200 record {|
-    string name?;
+public type inline_response_map200 record {
```

Comment:
Now we seem to generate types names (even for inline schemas) starting with simple letters, which is not aligned with Ballerina type naming conventions. Is this done intentionally?

Reply:
Yes, we have decided to generate (all) type names as it is in the openapi specification.

The type names for inline schemas are generated by the parser, and at the moment we are not special casing these. Moreover in the upcoming release, we will introduce a new option in the tool to disable this behaviour of the parser(the generated code will have inline records) - the default option may be the inline records

cc @lnash94

Reply:
Reference to this issue : https://github.com/ballerina-platform/ballerina-library/issues/5140 , as mentioned in @TharmiganK description we discussed moving forward with this change and will improve this name-changing option via in flatten feature